pom.xml

```diff
@@ -1217,6 +1217,7 @@
                                 <excludes combine.children="append">
                                     <!-- TODO: fix this in Airlift resolver -->
                                     <exclude>org.codehaus.plexus:plexus-utils</exclude>
+                                    <exclude>com.google.guava:guava</exclude>
```

Comment:
Just curious, what's this "TODO: fix this in Airlift resolver" is about?

Reply:
Not sure, that was there since November. @electrum, would you know about this?

Reply:
Oh, I misunderstood. Maybe we don't need to add guava to excludes here?

Reply:
That comment is specifically for `plexus-utils`. I don’t remember the details but should be fairly obvious if you remove it.

We do need the exclude for Guava.